spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -436,6 +470,19 @@ def main(argv=None):
              is_sct_binary=True,
              )
 
+    # un-straighten posterior disc map
+    # it won't exist if we don't use the detection since it is based on the network prediction (DL method)
+    if fname_disc is None and arguments.method == 'DL':
+        printv('\nUn-straighten posterior disc map...', verbose)
+        run_proc('sct_apply_transfo -i %s -d %s -w %s -o %s -x %s' %
+                 ('disc_posterior_tmp.nii.gz',
+                  'segmentation.nii',
+                  'warp_straight2curve.nii.gz',
+                  'label_disc_posterior.nii.gz',
+                  'label'),
+                 verbose=verbose
+                 )
```

Comment:
`run_proc` -> `sct_apply_transfo.main()`

Reply:
done in 4878610